website/docs/getting-started/setup/installation-guides/aws-ecs-on-fargate.md

```diff
@@ -18,41 +18,68 @@ This document guides you through installing Appsmith using AWS ECS on Fargate.
     Ensure that the EFS, ECS cluster, and Fargate instances are all in the same VPC.
     :::
       
-    3. Click on the EFS created and navigate to the Network tab to ensure mount-target is made in the same availability zone as the ECS cluster.
+    3. Click the EFS created and navigate to the Network tab to ensure the mount-target is made in the same availability zone as the ECS cluster.
     4. Create a new security group to allow inbound and outbound NFS traffic.
     5. Attach the security group to the EFS mount-target.
 
-5. **Create the ECS task execution role:**
+5. **Create the ECS task role:**
+    To enable the `exec` command, follow the steps below:
```

Comment:
```suggestion
```

Reply:
This is needed as suggested by Sumesh.

Reply:
Earlier, I wanted to highlight this as a note.

Reply:
The callout was not needed as it's information where the steps allow users to enable the `exec` command.